#!/usr/bin/env python

PACKAGE = "singaboat_vrx"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("gps_offset", double_t, 0, "GPS offset w.r.t. WAM-V along X-axis", 0.85, 0, 2.0)
gen.add("min_goal_tol", double_t, 0, "Minimum goal tolerance (for station-keeping)", 1.0, 0, 5.0)
gen.add("max_goal_tol", double_t, 0, "Maximum goal tolerance (for finding pinger)", 15.0, 0, 25.0)
gen.add("collision_tol", double_t, 0, "Obstacle range tolerance for collision avoidance", 17.5, 0, 25.0)
gen.add("collision_fov", double_t, 0, "Obstacle field of view for collision avoidance", 40.0, 0, 90.0)

exit(gen.generate(PACKAGE, "singaboat_vrx", "GymkhanaChallenge"))
